### ISSUE TITLE
Full Accessibility & 508 Office Audit [Sitewide Content - Contact Us page]

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#19401]
## Steps to complete the Full Accessibility & 508 Audit: 
- [x] VFS Product Manager: create issue and fill in feature name in the title and other bolded information appropriately
- [x] VFS: Link to this issue once created in [#vfs-platform-support](https://dsva.slack.com/channels/vfs-platform-support) in Slack; tag @ Trevor Pierce
- [ ] VSP: Accessibility specialists create sub-issues attached to this epic with findings
- [ ] VFS: Coordinate with the VA 508 Office to address accessibility findings
- [ ] VSP: [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/u/1/d/1d219oL1zCvCvnv1Bx-dI-GMzwgbarLv9_bzMSa3ULjA/edit#gid=1341642809) is updated

## Artifacts - _please provide the following artifacts when you request a full review_
- Access information for the tool or feature in staging _**do not put staging credentials in your `va.gov-team` ticket; store or reference them in a .md file in the `va.gov-team-sensitive` repository**_
- Link to verification that accessibility checks are complete:
  - (preferred) Link to your [TestRail VSP accessibility test plan template](https://dsvavsp.testrail.io/index.php?/suites/view/14&group_by=cases:section_id&group_order=asc).
OR
  - Link to a doc in your product folder that indicates all required checks from the accessibility [staging review prep](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md) doc are complete.
- List of known issues (with links to tickets when applicable)

Live page: https://www.va.gov/contact-us/
One known issue: #17984
More background in #17542

## Additional Notes
As part of this review, VSP will send your product to VA's 508 office for approval (as applicable)

**Comment by bethpottsVADEPO** [comment]
@1Copenut Can you confirm whether this ticket can be closed?

**Comment by shiragoodman** [comment]
hi @bethpottsVADEPO . Doesn't look like my team has gotten to this one yet. Otherwise, there would be accessibility issue tickets linked to it. Let me check with my team to see their availability on getting this done.

**Comment by noahgelman** [comment]
@bethpotts @shiragoodman I should have this done tomorrow

**Comment by noahgelman** [comment]
The accessibility review for this issue has been completed

**Comment by 1Copenut** [comment]
@RLHecht Please email the VA 508 office when you're ready for them to give a final quick test on the Contact Us page and CC Noah and Shira. DM us if you need a contact email address at 508. Thank you!

**Comment by bethpottsVADEPO** [comment]
@1Copenut Is this the process now or just with this root level, high-profile page? Just want to be clear for future: open ticket for internal accessibility check --> then you let us know if we need to get a 508 office review (or is this always the final step)?

**Comment by bethpottsVADEPO** [comment]
@1Copenut Also, can you provide the contact info for emailing the 508 office?

**Comment by FranECross** [comment]
This old epics tickets have been completed/closed; closing epic.
